test/fixtures/circular_deps_php_project/composer_deps.json

```diff
@@ -268,14 +395,29 @@
                           "php": {
                             "name": "php",
                             "version": "7.1",
-                            "dependencies": {}
+                            "dependencies": {},
+                            "labels": {
+                              "scope": "prod"
+                            }
                           }
+                        },
+                        "labels": {
```

Comment:
this looks weird, what are these emoty labels? Labels usually accompany a dep entry

Reply:
they have scope 'dev' or 'prod' to indicate whether the dependency is part of dev tree (require-dev) or prod tree (require).

is diff misleading? this label is on the "doctrine/instantiator" dependency and has scope "prod".